var/spack/repos/builtin/packages/sim-model/package.py

```diff
@@ -193,6 +194,9 @@ def _install_src(self, prefix):
 
     def _setup_build_environment_common(self, env):
         env.unset('LC_ALL')
+        # MPI flags should be the same as run environment, i.e. no wrappers
```

Comment:
```suggestion
        # MPI flags should be the same as run environment, i.e. no wrappers
        # TODO : check if OpenMPI recipe could be changed to avoid this explicit call
```

Reply:
Just for reference, this is common behavior of all MPI providers. For now, I find better to modify only one recipe, the one of the "strange" software. An alternative solution could however be something like removing from the environment vars MPIxx_CC MPIxx_CXX